src/main/java/school/lemon/changerequest/java/main/Main.java

```diff
@@ -0,0 +1,156 @@
+package school.lemon.changerequest.java.main;
+
+import school.lemon.changerequest.java.employees.*;
+import school.lemon.changerequest.java.matrix.Matrix;
+import school.lemon.changerequest.java.fractionnumber.FractionNumber;
+
+import java.util.Random;
+
+public class Main {
```

Comment:
You have 3 tasks and were asked to create 3 Demo classes.
Please split this class into 3 separate Demo classes. None of them should be `Main`, cause it is not a MAIN class.

They could be smth like `FractionNumberDemo`.